docs/lang/articles/reference/global_settings.md

```diff
@@ -9,11 +9,15 @@ sidebar_position: 3
 - To specify which kind of architecture (Arch) to use: `ti.init(arch=ti.cuda)`.
 - To specify the pre-allocated memory size for CUDA:
   `ti.init(device_memory_GB=0.5)`.
-- To specify which GPU to use for CUDA:
-  `export CUDA_VISIBLE_DEVICES=[gpuid]`.
+- To specify which GPU to use for CUDA(the first one is suggested):
+  `export TI_VISIBLE_DEVICE=[gpuid]` or `export CUDA_VISIBLE_DEVICES=[gpuid]`.
 - To disable a backend (`CUDA`, `METAL`, `OPENGL`) on start up, e.g. CUDA:
   `export TI_ENABLE_CUDA=0`.
 
+:::note
+
+If you set the value of one of the `TI_VISIBLE_DEVICE` and `CUDA_VISIBLE_DEVICES`, then the other's value will be the same. It's not suggested that you set different value of `TI_VISIBLE_DEVICE` and `CUDA_VISIBLE_DEVICES` which will raise a warning message.
```

Comment:
Setting either `TI_VISIBLE_DEVICE` or `CUDA_VISIBLE_DEVICES` will override the value of the other. Setting different values for `TI_VISIBLE_DEVICE` and `CUDA_VISIBLE_DEVICES` will likely cause undefined behavior thus not recommended.

Reply:
This is the old docs, I have updated it.

Reply:
Oh my bad